sentry_sdk/integrations/sanic.py

```diff
@@ -133,6 +135,7 @@ def _setup_sanic():
     # type: () -> None
     Sanic._startup = _startup
     ErrorHandler.lookup = _sentry_error_handler_lookup
+    # _patch_sanic_asgi()
```

Comment:
should probably removed

Reply:
Yes, I have this in my list of to-do items before merge